Defs/Ammo/Pistols/454Casull.xml

```diff
@@ -102,7 +102,7 @@
 		<label>.454 Casull bullet (FMJ)</label>
 		<projectile Class="CombatExtended.ProjectilePropertiesCE">
 			<damageAmountBase>20</damageAmountBase>
-			<armorPenetrationSharp>6</armorPenetrationSharp>
```

Comment:
What info is this stat changes based upon?

Reply:
Realism reasons based on gel penetration data and more proper scaling